CONTRIBUTING.md

```diff
@@ -65,6 +65,10 @@ Other Tips
 
         $ git commit --amend
 
+  - Then force push your branch:
+
+        `$ git push origin branch_name -f`
```

Comment:
> This doesn't need the backticks around it--indented blocks are already printed monospaced.

I tried it intended, and looks like it fails to pick it up. I can do that once more, for you to review.



> Also I would rather spell this git push --force origin branch_name; it makes it more clear that --force is an argument to git push. I think git push --force also needs to be used carefully, and I usually try to make myself spell it out in order to give myself more time to think about what I'm doing--good practice to promote.

Yes, makes sense.